### PR TITLE
add test for `anti_flood_ftpd.py` demo script.

### PR DESCRIPTION
**Pull request by hramezani**
I also add tests for other demo scripts soon.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -106,6 +106,9 @@ test-ioloop: install
 test-servers: install
 	$(PYTHON) pyftpdlib/test/test_servers.py
 
+test-servers: install
+	$(PYTHON) pyftpdlib/test/test_demo_scripts.py
+
```
Comment (giampaolo) [inline Makefile]: There is already a make command with this name. You should rename this to `test-demos`

File: pyftpdlib/test/__init__.py [hunk 2]
```diff
@@ -362,9 +362,11 @@ class ThreadedTestFTPd(ThreadWorker):
     shutdown_after = 10
     poll_interval = 0.001 if TRAVIS else 0.000001
 
-    def __init__(self, addr=None):
+    def __init__(self, addr=None, handler=None):
         super(ThreadedTestFTPd, self).__init__(
             poll_interval=self.poll_interval)
+        if handler:
+            self.handler = handler
```
Comment (giampaolo) [inline pyftpdlib/test/__init__.py]: I prefer not to do this. You can simply `setattr` the handler later against this object

**Comment by hramezani** [comment]
@giampaolo  i fix the problems. i change the `__init__` of  `ThreadedTestFTPd`. do you like it?

**Comment by hramezani** [comment]
@giampaolo any update?

**Pull request closed without merging**